Detectors/TRD/simulation/src/Digitizer.cxx

```diff
@@ -70,6 +70,23 @@ void Digitizer::init()
   }
 
   mSDigits = false;
+  setupSimulationValues();
+}
+
+void Digitizer::setupSimulationValues()
+{
+  kNpad = mSimParam->getNumberOfPadsInPadResponse(); // Number of pads included in the pad response
+  kAmWidth = TRDGeometry::amThick();                 // Width of the amplification region
+  kDrWidth = TRDGeometry::drThick();                 // Width of the drift retion
+  kDrMin = -0.5 * kAmWidth;                          // Drift + Amplification region
+  kDrMax = kDrWidth + 0.5 * kAmWidth;                // Drift + Amplification region
+  timeBinTRFend = 0;
+  if (mSimParam->TRFOn()) {
+    timeBinTRFend = ((int)(mSimParam->GetTRFhi() * mCommonParam->GetSamplingFrequency())) - 1;
+  }
+  nTimeTotal = kTimeBins; // PLEASE FIX ME when CCDB is ready
```

Comment:
same comment as above.

Reply:
This is a good catch, indeed. The intend of the variable is not clear. Even more, it is not clear to me is this should be defined in `TRDCommonParams.h` or `TRDSimParams.h`; maybe @tdietel can comment, too.